app/services/notices/setup/determine-return-forms.service.js

```diff
@@ -16,7 +14,7 @@ const ReturnFormsNotificationPresenter = require('../../../presenters/notices/se
  * @param {object[]} recipients - List of recipient objects, each containing recipient details like email / address.
  * @param {string} eventId - the event id to link all the notifications to an event
  *
- * @returns {Promise<object[]>} - Resolves an array of return forms notifications
+ * @returns {object[]} - Resolves an array of return forms notifications
```

Comment:
If this is no longer an async service then we should remove `async` from `go()`. Else this change needs to go back in.